portal-web/test/functional/com/liferay/portalweb/macros/AdaptiveMedia.macro

```diff
@@ -212,6 +212,7 @@
 			</else>
 		</if>
 
+		<execute function="AssertClick" locator1="Button#SAVE" value1="Save" />
```

Comment:
Hey @acbbennett Why did you add this line? This is covered in `PortletEntry#save`